docs/README.md

```diff
@@ -1,10 +1,14 @@
 ## docs for docs!
 
-At Tidepool, we are starting to use [GitBook](https://www.gitbook.com/) for documentation across our client-side apps and here in this repository for "no man's land" docs without a proper home in another repo. We use [GitHub Pages](https://pages.github.com/) to publish the GitBook-generated docs on the web at various paths under the base URL of our developer microsite at [http://developer.tidepool.io/](http://developer.tidepool.io/). If you need to add or update the docs in this repo or in another repo that employs GitBook for publishing documentation, follow the instructions below.
+At Tidepool, we are using [GitBook](https://www.gitbook.com/) for documentation across our client-side apps and here in this repository for "no man's land" docs without a proper home in another repo. GitBook is a tool for generating a static documentation website (with built-in search) from a set of Markdown files (which are handily thus natively renderable by the GitHub online interface). We use [GitHub Pages](https://pages.github.com/) to publish the GitBook-generated static assets on the web at various paths under the base URL of our developer microsite at [http://developer.tidepool.io/](http://developer.tidepool.io/). If you need to add or update the docs in this repo or in another repo that employs GitBook for publishing documentation, follow the instructions below.
 
 ### Setting up your repo to work on docs
 
-When you use GitHub Pages to host a website for a particular project (like this one), you must create a branch titled `gh-pages` that contains all the files for your (static) website. Since for most repositories, this `gh-pages` branch contains content that is very different from other branches, [GitHub's advice](https://help.github.com/articles/creating-project-pages-manually/) is to use a separate clone of the repository to manage *just* the `gh-pages` branch. This is what jebeck does, and she recommends the following set up procedure:
+When we at Tidepool first set up our docs workflow using GitBook, there was only one way to set up [GitHub Pages](https://pages.github.com/ 'GitHub Pages') for a project: putting the static assets (HTML, CSS, etc.) you wanted GitHub to serve as your GitHub Pages site on a branch in your repository named `gh-pages`. This method of building a GitHub Pages site still exists, and it's what we still use at Tidepool[^a]. Every time we update the static assets on the `gh-pages` branch and push to the GitHub remote repository, our project page is updated to serve the new static files.
```

Comment:
That doesn't work with GitBook, unfortunately. But GitBook creates great footnote links with the native Markdown.